ozaria/site/components/teacher-dashboard/modals/ModalGetLicenses.vue

```diff
@@ -278,6 +279,12 @@ export default Vue.extend({
   max-width: 600px;
 }
 
+.form-error {
+  color: #a94442;
```

Comment:
you can use: `$color-concept-flag-color`, it will be similar to red. We have lot of code where we directly put hex color but we should be ideally using variables for them and use from a limited set of colors to maintain consistency